uwsgi.h

```diff
@@ -1356,8 +1356,11 @@ struct wsgi_request {
 	char *appid;
 	uint16_t appid_len;
 
-	//this is big enough to contain sockaddr_in
-	struct sockaddr_un c_addr;
+	union address {
```

Comment:
Just to confirm, you want me to leave the existing type-punned `c_addr` in place and just not use it? I want to make sure I make the right change.